### ISSUE TITLE
tests/cython cxxtests are broken

### ISSUE THREAD

**Issue by vsbogd** [opencog/atomspace#1801]
Few issues here:
1. tests/CMakeLists.txt checks PYTHONINTERP_FOUND before including cython tests folder:
```
	IF (HAVE_CYTHON AND PYTHONINTERP_FOUND)
		ADD_SUBDIRECTORY (cython)
	ENDIF (HAVE_CYTHON AND PYTHONINTERP_FOUND)
```
But PYTHONINTERP_FOUND is unset in lib/FindCxxtest.make and lib/FindPython3Interp.cmake so cython folder is never included into list of unit tests.

In particular ```make test_python``` command fails:
```
make[3]: *** No rule to make target 'atomspace/PythonEvalUTest', needed by 'CMakeFiles/test_python'.  Stop.
CMakeFiles/Makefile2:279: recipe for target 'CMakeFiles/test_python.dir/all' failed
make[2]: *** [CMakeFiles/test_python.dir/all] Error 2
CMakeFiles/Makefile2:286: recipe for target 'CMakeFiles/test_python.dir/rule' failed
make[1]: *** [CMakeFiles/test_python.dir/rule] Error 2
Makefile:225: recipe for target 'test_python' failed
```

2. Tests in tests/cython folder fails:
```
57% tests passed, 3 tests failed out of 7

Total Test time (real) =   4.25 sec

The following tests FAILED:
	  1 - PythonEvalUTest (Child aborted)
	  4 - CythonAtomSpace (Failed)
	  7 - CythonGuile (Failed)
```

**Comment by linas** [comment]
FYI, for me, running Debian stable, and having installed whatever it is I have installed, all tests pass 100% of the time, more or less always, and its been like that for maybe a year now.

**Comment by vsbogd** [comment]
Yes, ```make test``` passes but C++ Python related unit tests are not started. This is a subject of the issue.

**Comment by linas** [comment]
Sorry, please ignore my earlier comment. yes, `make test_python` fails for me with same error as above.  In addition, for me, the `tests/cython` directory is never even visited, the tests are never even built. 

I got very used to seeing these tests run and pass, that I never noticed when they stopped getting run.  I'm thinking that the test report should include, at the bottom, a note "the python tests were not run", and for the sql tests, "the sql tests were not run", (as those are the other optional set of unit tests)

**Comment by noskill** [comment]
I made a new issue for remaining test https://github.com/opencog/atomspace/issues/1888

**Comment by linas** [comment]
I just now typed `make test_python` and so it appears that this is now fixed; it must have been fixed in #1887  -- can this bug now be  closed?
